### PR TITLE
example pull request - introducing a concatenated code

### PR DESCRIPTION
**Pull request by phfaist**

### CODE REVIEW

File: codes/qubits/concatenated_code.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+protection: 'Protection depends on the chosen codes.'
+
+#features:
+  #gates: 'Keep it succinct for now'
+  #fault_tolerance: 'If applicable, any fault-tolerant protocols with the code.'
+  #threshold: 'Numerical threshold of 2% \cite{only arxiv:### or doi:###}. Concatenated-scheme threshold of ...'
+  #make one that you feel is applicable to a large and interesting class of codes
+  #encoder: ''
+
+notes: '.'
```
Comment (valbert4) [inline codes/qubits/concatenated_code.yml:18]: test

File: codes/qubits/concatenated_code.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# UTF-8 unicode encoding
+
+code_id: concatenated_code
+
+name: 'Concatenated code'
+
+description: 'A concatenated code is a code constrcuted from two error-correcting codes, where the logical information is encoded using a first code, and the encoded result is further encoded using the second code.'
+
+protection: 'Protection depends on the chosen codes.'
```
Comment (valbert4) [inline codes/qubits/concatenated_code.yml:9]: test 2

File: codes/qubits/concatenated_code.yml (hunk 1 above)
Comment (phfaist) [reply codes/qubits/concatenated_code.yml:18]: nice!

File: codes/qubits/concatenated_code.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# UTF-8 unicode encoding
+
+code_id: concatenated_code
+
+name: 'Concatenated code'
+
+description: 'A concatenated code is a code constrcuted from two error-correcting codes, where the logical information is encoded using a first code, and the encoded result is further encoded using the second code.'
```
Comment (valbert4) [inline codes/qubits/concatenated_code.yml:7]: please add...

**Pull request closed without merging**